components/raftstore/src/store/fsm/peer.rs

```diff
@@ -3212,10 +3234,17 @@ where
                     }
                 }
             }
-        } else {
-            self.fsm
-                .peer
-                .execute_transfer_leader(self.ctx, msg.get_from(), peer_disk_usage, false);
+        } else if !self
+            .fsm
+            .peer
+            .maybe_reject_transfer_leader_msg(self.ctx, msg, peer_disk_usage)
+        {
+            // If max warmup time is 0, then skip the warmup process and ack directly.
+            if self.ctx.cfg.max_raft_entry_cache_warmup_time.0 == Duration::from_secs(0)
```

Comment:
it can be wrapped into a inline "fn pre_warmup_entry_cache_enabled() -> bool" for better readability.

Reply:
fixed